lib/Sema/DerivedConformanceCodable.cpp

```diff
@@ -353,8 +358,21 @@ static EnumDecl *synthesizeCodingKeysEnum(DerivedConformance &derived) {
       }
 
       case DoesNotConform:
+        // for accurate diagnostic, we show TypeRepr if present for IUO
+        TypeLoc typeLoc = TypeLoc();
+        if(varDecl->isImplicitlyUnwrappedOptional()){
```

Comment:
```suggestion
        if(varDecl->isImplicitlyUnwrappedOptional()) {
```

Reply:
In general, please run `clang-format` on this buffer.